products/fuse/get-started.adoc

```diff
@@ -33,7 +33,7 @@ For Linux Development Hosts:
 Go to the folder where you have downloaded JBoss Developer Studio, and execute the
 installer +
 +
-`java -jar jboss-devstudio-8.1.0.GA-installer-standalone.jar`
+`java -jar jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar`
```

Comment:
This should be changed the 8.1.0 eap bundle jar, not the 9.0 one.

https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-jar_universal.jar